org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/RolePanel.java

```diff
@@ -563,6 +574,25 @@ else if (ON_DEFER_LOGOUT.equals(event.getName()))
 		}
     }
     
+	private void openLoginHelp() {
+		String lang = Env.getContext(Env.getCtx(), Env.LANGUAGE);
+		lang = lang.substring(0, 2);
+		String helpURL = MSysConfig.getValue(MSysConfig.LOGIN_SELECTION_HELP_URL, "https://wiki.idempiere.org/{lang}/Login_Selection_Help");
```

Comment:
It looks a bit strange to that that the title of the dialog/page is "Select Role" but the help page title is "Login Selection".